c_src/exml_event.c

```diff
@@ -156,22 +204,23 @@ static ERL_NIF_TERM new_parser(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv
     return enif_make_tuple(env, 2, OK, parser_resource);
 };
 
+static void reset_parser_int(ErlNifEnv *env, XML_Parser parser)
```

Comment:
Sure, but in the same vein we could rename `reset_parser` function to `reset_and_init_parser`. I wanted to highlight that `reset_parser` is just a wrapper for `reset_parser_int` which is internally used from more funcitons; the other way I could do it is renaming them to `reset_parser` (internal) and `reset_parser_nif` (wrapper).